index.html

```diff
@@ -10,66 +10,144 @@
   <main>
   <h1>IPFS <br>Implementations <br>Grants</h1>
   <p>
-    The IPFS Implementations Grants program was established in 2022 to advance the development, growth, and impact 
+    The IPFS Implementations Grants program was established in 2022 to advance the development, growth, and impact
     of the InterPlanetary File System (IPFS) project through a focus on developer choice and availability. We provide
-    financial support to projects and teams working on integrations, extensions, and new implementations that make 
+    financial support to projects and teams working on integrations, extensions, and new implementations that make
     IPFS accessible to more developer communities across a variety of languages, platforms, and systems.
   </p>
+
+  <h2>Spring 2025 Request for Proposals: Content-Addressed Data Tools</h2>
+  <p>The IPFS Implementations Fund supports a more open, resilient, and efficient web by funding  implementations of the <a href="https://specs.ipfs.tech/">IPFS protocol</a>. It aims to broaden the reach and impact of the IPFS project by making content addressing and peer-to-peer networking more accessible across different platforms, programming languages, environments, and use cases.</p>
+  <p>Our Utility Grants program supports developers creating essential utilities, libraries, and tooling. Grants of $5,000-25,000 per project aim to fill critical tooling gaps and strengthen the foundations of an open and interoperable web.</p>
+  <p>For the Spring 2025 cycle, we seek proposals from qualified individuals or teams for the following:</p>
+
+  <h3>RFP #2025-01: CAR Archive Explorers and Utilities</h3>
+  <p>The CAR format (Content Addressable aRchives) makes data reliably portable by packaging up content with its cryptographic identifiers. This ensures the same data can be verified and used across different systems and platforms, making it a good building block for “credible exit” and data portability. Typically found in a file with a <code>.car</code> filename extension, you can think of them like TAR files that are designed for storing collections of content addressed data.</p>
+  <p>Originally created as a way to create stable documents from large or dynamic IPLD graphs (including those graphs that populate “virtual directories” of files added to IPFS), CAR files can now be found in IPFS-based systems, Storacha, ATProto, and other open source projects. With growing interest in data independence – moving large datasets between storage providers, backing up personal content from hosted services, and maintaining data integrity across the web – there's a growing need for accessible tools that help people create, inspect, and manage their own CAR files.</p>
+  <p>Currently, there is limited tooling for the CAR format. Prior art includes Bluesky’s <a href="https://github.com/bluesky-social/indigo/blob/main/cmd/gosky/car.go">car.go</a>, which offers subcommands to work with CAR files on local disk.</p>
```

Comment:
Version control! This link was changed recently.

```suggestion
  <p>Currently, there is limited stand-alone tooling for the CAR format. Prior art includes Bluesky’s <a href="https://github.com/bluesky-social/indigo/blob/main/cmd/goat/repo.go">goat</a>, but this is specific to BlueSky-export .car files and not ergonomic to use outside of that context.</p>
  
```